### ISSUE TITLE
getReact() seem not to work properly with immutablejs?

### ISSUE THREAD

**Issue by thomaspersson** [DevExpress/testcafe-react-selectors#26]
**tl;dr;**
**Doing getReact().props.foo where foo is a List() (immutableJS) the toJS and toArray Iterable Prototype functions are missing.** 

Therere seem to be a problem with [ImmutableJS](https://facebook.github.io/immutable-js/) and testcafe-react-selector.

I have a component called UserList that has prop "users" which is an List (immutablejs structure). If I do 

`const listComp = await ReactSelector('App UserList');`
`const compData = await listComp.getReact();`
`const users = compData.props.users;`

I should be able to either todo users.toJS() or users.toArray() but I get an error saying "toArray/toJS is not a function..." and if I do
`
console.log(users)
`
It will return 
`
{ size: 10,
     _origin: 0,
     _capacity: 10,
     _level: 5,
     _root: null,
     _tail: { array: [Object], ownerID: undefined },
     __ownerID: undefined,
     __hash: undefined,
     __altered: false
}
`
If I print out that property in the component and watch in Safari I can see that toJS and toArray is part of the objects 'Iterable Prototype'. Are you maybe copying the object but does not copying the prototype fully?

**Comment by anshumanbora** [comment]
Did someone find any solution to this issue? I'm facing the same issue.

**Comment by kirovboris** [comment]
We're going to add an API for creating TestCafe Live Runner in the next major release. Also TestCafe Live will be included in the main testcafe package since `testcafe@1.0.0`.

**Comment by kirovboris** [comment]
`await getReact(...)` serializes the props and sends them to the testcafe server. We can't execute the `toJS` function in the nodejs context, but you can try to use the following approach
```js
const listComp = await ReactSelector('App UserList');
const users = await listComp.getReact(({props}) => props.users.toJS());
```
In this case the filter function executes in the browser context.
